### DISCUSSION TITLE
Disable "Execution finished" dialog?

### DISCUSSION THREAD

**Discussion by markjuggles** [chriskiehl/Gooey#898] [Q&A] [answered]
Is there a way to disable the "Execution finished" dialog?

I would think that printing the results in the Status TextCtrl would be sufficient.

**Comment by bgmwaring** [comment, marked as answer]
Add the follow to your GOOEY header.
```p
show_success_modal=False
```

**Reply by markjuggles** [reply]
Perfect, thank you!
